Ryujinx.HLE/HOS/Services/Ssl/SslService/ISslContext.cs

```diff
@@ -1,7 +1,25 @@
+using Ryujinx.Common.Logging;
+using System;
+
 namespace Ryujinx.HLE.HOS.Services.Ssl.SslService
 {
     class ISslContext : IpcService
     {
-        public ISslContext() { }
+        public ISslContext(ServiceCtx context ) { }
+        
+        [Command(4)]
+        public ResultCode ImportServerPki(ServiceCtx context)
+        {
+            var CertificateFormat = context.RequestData.ReadInt32();
+            var unknown = context.RequestData.ReadBytes(5);
```

Comment:
As defined on [SwIPC](https://reswitched.github.io/SwIPC/ifaces.html#nn::ssl::sf::ISslContext(4)), there isn't 5 bytes here but an IPC send buffer:
```suggestion
            long certificateDataPosition = context.Request.SendBuff[0].Position;
            long certificateDataSize     = context.Request.SendBuff[0].Size;
```